V/VerizonEdgecastTokenBuilder/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "VerizonEdgecastTokenBuilder"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    "https://github.com/VerizonDigital/ectoken.git" =>
+    "7b8812d476f5be5b290fe2832859b9b7636f43ae",
+
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+export OPENSSL_INCLUDE="-I $prefix/include"
+export OPENSSL_LIBS="-lssl -lcrypto"
+cd ectoken/c-ectoken/ecencrypt/
+gcc -m64 -O2 -Wall -Werror -std=gnu99 ec_encrypt.c ectoken_v3.c base64.c -o 64/ectoken3 $OPENSSL_LIBS $OPENSSL_INCLUDE -lm -lpthread -ldl
+cp 64/ectoken3 $prefix/bin/
+exit
+
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:x86_64, libc=:glibc)
+]
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("ectoken3", :ectoken3)
```

Comment:
Do you feel like it's important to have the number in the wrapper function name? BTW, what does the number stand for? Is it a version number?

Reply:
It looks like upstream calls it `ectoken3`.  So perhaps that should be the overall name of the entire project?

Reply:
Yes, 3 is likely a version number. Just `ectoken` may not be clear as to what it is. But maybe it's okay as it is not like a regular Julia package that someone wants to discover?

I am open to shortening the name to one of - `VerizonEdgecaseToken`, `VerizonEctoken`, or just `ectoken`. Is there any preference?

Reply:
I was going to say simply "ectoken", but I found this [other project with the same name](https://github.com/soundcloud/ectoken) and similar goal, as far as I can tell.  Maybe "VerizonEctoken"?

Reply:
Sounds good :+1:. I'll update the PR.